### ISSUE TITLE
UniLM使用electra，loss为NAN

### ISSUE THREAD

**Issue by fuyitao** [bojone/bert4keras#167]
提问时请尽可能提供如下信息：

### 基本信息
- 你使用的**操作系统**: linux
- 你使用的**Python**版本: 3.6
- 你使用的**Tensorflow**版本: :tensorflow-gpu==1.13.1
- 你使用的**Keras**版本: keras==2.2.4
- 你使用的**bert4keras**版本: 0.7.7
- 你使用纯**keras**还是**tf.keras**: keras
- 你加载的**预训练模型**: chinese_electra_base_L-12_H-768_A-12/

### 核心代码
model = build_transformer_model(
     config_path,
     checkpoint_path,
     model='electra',
     application='unilm',
     keep_tokens=keep_tokens,  # 只保留keep_tokens中的字，精简原字表
)

### 输出信息
Transformer-11-MultiHeadSelfAtt (None, None, 768)    1536        Transformer-11-MultiHeadSelfAtten
__________________________________________________________________________________________________
Transformer-11-FeedForward (Fee (None, None, 768)    4722432     Transformer-11-MultiHeadSelfAtten
__________________________________________________________________________________________________
Transformer-11-FeedForward-Drop (None, None, 768)    0           Transformer-11-FeedForward[0][0] 
__________________________________________________________________________________________________
Transformer-11-FeedForward-Add  (None, None, 768)    0           Transformer-11-MultiHeadSelfAtten
                                                                 Transformer-11-FeedForward-Dropou
__________________________________________________________________________________________________
Transformer-11-FeedForward-Norm (None, None, 768)    1536        Transformer-11-FeedForward-Add[0]
==================================================================================================
Total params: 101,677,056
Trainable params: 101,677,056
Non-trainable params: 0
__________________________________________________________________________________________________
Instructions for updating:
Use tf.cast instead.
Epoch 1/50
2020-06-12 09:21:34.970066: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcublas.so.9.0 locally
 154/1194 [==>...........................] - ETA: 31:01 - loss: nan

### 自我尝试
1. 在 “chinese_roberta_wwm_ext_L-12_H-768_A-12” 的预训练模型上训练一直没有问题，将model改成'electra'就出现loss=nan了，模型其他处没有修改
2. 之前使用electra做序列标注是可以训练执行的

**Comment by bojone** [comment]
electra不能用于语言模型相关的（lm、unilm都不行）。是我没提示充分，下一个版本加上～

**Comment by fuyitao** [comment]
好的，明白
